File: src/main/java/me/miki/shindo/management/event/impl/EventLivingUpdate.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;
import net.minecraft.entity.EntityLivingBase;

public class EventLivingUpdate extends Event {

    private final EntityLivingBase entity;

    public EventLivingUpdate(EntityLivingBase entity) {
        this.entity = entity;
    }

    public EntityLivingBase getEntity() {
        return entity;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventLivingUpdate.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event
import net.minecraft.entity.EntityLivingBase

class EventLivingUpdate(
    private val _entity: EntityLivingBase,
) : Event() {
    fun getEntity(): EntityLivingBase = _entity
}
